tests/src/test/scala/com/nvidia/spark/rapids/AdaptiveQueryExecSuite.scala

```diff
@@ -85,7 +85,7 @@ class AdaptiveQueryExecSuite
 
   private def findTopLevelGpuBroadcastHashJoin(plan: SparkPlan): Seq[GpuExec] = {
     collect(plan) {
-      case j: GpuExec if ShimLoader.getSparkShims.isBroadcastExchangeLike(j) => j
+      case j: GpuExec if j.isInstanceOf[BroadcastExchangeLike] => j
```

Comment:
nit: why not just
```
case j: BroadcastExchangeLike => j
```

Reply:
That doesn't compile since it needs to return a sequence of `GpuExec` but `BroadcastExchangeLike` does not implement that trait. I do think I can simplify it a bit by using `with BroadcastExchangeLike` instead of the `isInstanceOf` check.